vars/getBaseVersionFromPackageJson.groovy

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/groovy
+
+import groovy.json.JsonSlurper
```

Comment:
We seem to use `JsonSlurperClassic` in some places. Not sure if there's a specific reason for that, is there? This seems to work fine here at least.

Reply:
https://stackoverflow.com/questions/37864542/jenkins-pipeline-notserializableexception-groovy-json-internal-lazymap